File: sources/Old-Raze/event/impl/PacketEvent.java
package markgg.event.impl;

import markgg.event.Event;
import net.minecraft.network.Packet;

public class PacketEvent extends Event {

    public static Packet packet;

    public PacketEvent(Packet packet) {
        this.packet = packet;
    }

    public static Packet getPacket() {
        return packet;
    }
}

File: sources/Leaf devEnv/com/leafclient/leaf/event/game/network/PacketEvent.kt
package com.leafclient.leaf.event.game.network

import fr.shyrogan.publisher4k.Cancellable
import net.minecraft.network.Packet

abstract class PacketEvent(var packet: Packet<*>): Cancellable() {

    class Send(packet: Packet<*>): PacketEvent(packet)

    class Receive(packet: Packet<*>): PacketEvent(packet)

}